install_deps.sh

```diff
@@ -24,7 +26,7 @@ then
         git clone --branch 3.4.6 https://github.com/opencv/opencv/
         git clone --branch 3.4.6 https://github.com/opencv/opencv_contrib/
         mkdir -p opencv/build && cd opencv/build
-        cmake -DOPENCV_EXTRA_MODULES_PATH=../../opencv_contrib/modules ..
+        cmake -D CMAKE_BUILD_TYPE=Release -D CMAKE_INSTALL_PREFIX=/usr/local -DBUILD_TESTS=OFF -DBUILD_PERF_TESTS=OFF -DBUILD_EXAMPLES=OFF -DBUILD_JAVA=OFF -DWITH_OPENGL=ON -DOPENCV_EXTRA_MODULES_PATH=../../opencv_contrib/modules ..
```

Comment:
Nit: Do we let the user's know that we are installing opencv into their root directory?

Reply:
Install deps now asks for permission before modifying the user's system.